common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -583,3 +583,20 @@ def index_dictionary(self):
         xblock_body["content_type"] = "Sequence"
 
         return xblock_body
+
+
+class SummaryFields(object):
+    """Only Sections have summaries now, but we may expand that later."""
+    summary = List(
+        help=_("A list summarizing what students should look forward to in this section."),
+        scope=Scope.settings
+    )
+
+
+class SectionModule(SummaryFields, SequenceModule):
+    """Module for a Section/Chapter."""
+
+
+class SectionDescriptor(SummaryFields, SequenceDescriptor):
```

Comment:
Does this need to be used somewhere?  Or did you plan to do so in a subsequent PR?

Reply:
nvm.  I see the mapping is just in setup.py.  Perhaps that's why it didn't work on your devstack before.  I also have had issues in the past with setup.py not running when expected in docker.